webapp/store/search.js

```diff
@@ -46,8 +46,8 @@ export const actions = {
     await this.app.apolloProvider.defaultClient
       .query({
         query: gql`
-          query findPosts($query: String!) {
-            findPosts(query: $query, limit: 10) {
```

Comment:
I find `query` a better naming for a string that is used for full-text search.

Reply:
I could agree with that, but if it's named query it breaks the functionality. not sure why exactly cause I'm not sure I completely understand the flow here, but it seems it passes through the `schema.gql` where we use the magic here https://github.com/Human-Connection/Human-Connection/blob/master/backend/src/schema/types/schema.gql#L10
then it gets passed to the `posts` resolver, which calls the `filterForBlockedUsers`
then cause there are no `badIds` it returns the `params`, otherwise it adds them to the `filter`, but the posts which match the query have already been returned from the database :point_up: